index.js

```diff
@@ -42,6 +48,9 @@ const openNewBrowserInTheSamePage = function(browser) {
 
 const isCurrentUrlDifferentFromBaseUrl = function() {
   return browser.getCurrentUrl().then(url => {
+    console.log(url);
```

Comment:
Why these console.logs?

Reply:
Tt was only for my tests, I forgot to remove.